### PR TITLE
ARSN-162: add getBucketTagging NoSuchTagSet error

### PR DESCRIPTION
**Pull request by ghost**
Necessary for proper function of GetBucketTagging API.
Ref: https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetBucketTagging.html

### CODE REVIEW

**Review by miniscruff — APPROVED** [review]

File: lib/errors/arsenalErrors.ts [hunk 1]
```diff
@@ -365,6 +365,11 @@ export const NoSuchWebsiteConfiguration: ErrorFormat = {
     description: 'The specified bucket does not have a website configuration',
 };
 
+export const NoSuchTagSet: ErrorFormat = {
+    code: 404,
+    description: 'The TagSet does not exist',
```
Comment (ghost) [inline lib/errors/arsenalErrors.ts:370]: Why not:

```suggestion
    description: 'There is no tag set associated with the bucket.',
```

To be standard with the AWS error description? :smile:

**Review by ghost — APPROVED** [review]

File: lib/errors/arsenalErrors.ts (hunk 1 above)
Comment (ghost) [reply lib/errors/arsenalErrors.ts:370]: As mentioned in PM, the new message is 'The TagSet does not exist'
```
aws s3api get-bucket-tagging --bucket anuragplatform

An error occurred (NoSuchTagSet) when calling the GetBucketTagging operation: The TagSet does not exist
```

**Comment by ghost** [comment]
ping

**Review by anurag4DSB — APPROVED** [review]

**Comment by anurag4DSB** [comment]
Integration PR looks good to me.

**Pull request merged**